datadog/resource_datadog_dashboard.go

```diff
@@ -4917,6 +4933,156 @@ func buildTerraformServiceLevelObjectiveDefinition(datadogDefinition datadogV1.S
 	return terraformDefinition
 }
 
+//
+// SLO (Service Level Objective) List Widget Definition helpers
+//
+
+func getSloListDefinitionSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"request": {
```

Comment:
is this `request ` or `requests`? asking as it is listed as `requests` in our doc page - https://docs.datadoghq.com/dashboards/widgets/slo_list/#api

Reply:
Great question, you're right that it's named `requests` on the API side. The reason why we use the singular form here is due to how lists are defined in terraform. For example, if you wanted to specify multiple request objects your terraform would look something like this:
```
resource "datadog_dashboard" "slo_list_dashboard" {
	title         = "{{uniq}}"
	description   = "Created using the Datadog provider in Terraform"
	layout_type   = "ordered"
	is_read_only  = "true"

	widget {
		slo_list_definition {
			request {
				request_type = "slo_list"
				query {
					query_string = "query one"
					limit = 30
				}
			}
			request {
				request_type = "slo_list"
				query {
					query_string = "query two"
					limit = 30
				}
			}
			request {
				request_type = "slo_list"
				query {
					query_string = "query N"
					limit = 30
				}
			}
			...
			title = "my title"
			title_size = "16"
			title_align = "center"
		}
	}
}
```
_(note: the slo list widget does not support multiple requests, the example above is just for demonstration purposes)_

If we used the plural form, it would look like you're overwriting the requests array with every block, when what you're actually doing is appending to it

Reply:
Thanks for the example! all good now! got confused when I read the doc page but then answered myself after read the build method below again :)